src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -266,5 +266,24 @@ public void GetFullPath_CoreTests(string path, string expected)
         {
             Assert.Equal(expected, Path.GetFullPath(path));
         }
+
+        [Theory,
+        InlineData(@"C:\folder\", @"C:\folder"),
+        InlineData(@"C:/folder/", @"C:/folder"),
+        InlineData(@"/folder/", @"/folder"),
+        InlineData(@"\folder\", @"\folder"),
+        InlineData(@"/folder//", @"/folder/"),
+        InlineData(@"\folder\\", @"\folder\"),
+        InlineData(@"C:\", @"C:\"),
+        InlineData(@"C:/", @"C:/"),
+        InlineData(@"", @""),        
+        InlineData(@"/", @"/"),
+        InlineData(@"\", @"\"),
+        InlineData(null, null)]
+        public void TrimEndingDirectorySeparator_CoreTests(string path, string expected)
+        {
+            Assert.Equal(expected, Path.TrimEndingDirectorySeparator(path));
+            PathAssert.Equal(expected, Path.TrimEndingDirectorySeparator(path.AsSpan()));
+        }
```

Comment:
What about the span overload?

Reply:
I don't understand, second assert is for ROS overload `PathAssert.Equal(expected, Path.TrimEndingDirectorySeparator(path.AsSpan()));`

Reply:
Sorry, I totally didn't see it.

Reply:
done